docs/components/connectors/out-of-the-box-connectors/aws-sns-inbound.md

```diff
@@ -0,0 +1,156 @@
+---
+id: aws-sns-inbound
+title: Amazon Simple Notification Inbound Service Connector
```

Comment:
```suggestion
title: Amazon Simple Notification Service Inbound Connector
```
To keep the product name (SNS) together.

Reply:
Done